components/organisms/footer/footer.js

```diff
@@ -72,10 +76,11 @@ const StyledFooter = styled.div`
 
 const StyledList = styled(List)`
   display: flex;
-  justify-content: flex-end;
```

Comment:
Add flex-direction: column, so that list will be vertical. Or just edit display: flex to inline?

Reply:
@zemqi Yeah thats a larger fix - wanted it to work with Cleverly and Portal right now then I will fix the lists up better. They need a refactor. If happy with explanation please merge